.travis.yml

```diff
@@ -1,9 +1,12 @@
 language: ruby
+sudo: false
```

Comment:
We might require sudo here - all the tests are failing. Can you try removing this line?